### ISSUE TITLE
Create Vagrant env for this tool

### ISSUE THREAD

**Issue by ShahriyarR** [ShahriyarR/MySQL-AutoXtraBackup#115]
For simplifying our life, it will be great if we have vagrant

**Comment by ShahriyarR** [comment]
so the result is, I am using centos/7 in vagrant and could install `autoxtrabackup` while provisioning:

```
[vagrant@localhost ~]$ autoxtrabackup --version
Developed by Shahriyar Rzayev from Azerbaijan MUG(http://mysql.az)
Link : https://github.com/ShahriyarR/MySQL-AutoXtraBackup
Email: rzayev.shahriyar@yandex.com
Based on Percona XtraBackup: https://github.com/percona/percona-xtrabackup/
MySQL-AutoXtraBackup Version: 1.4.8
```

**Comment by ShahriyarR** [comment]
But I need to test it further, to publish.

**Comment by ShahriyarR** [comment]
This thing is implemented and will be available on 1.4.9 release.
